sys/net/gnrc/routing/rpl/gnrc_rpl_dodag.c

```diff
@@ -276,10 +341,9 @@ void gnrc_rpl_local_repair(gnrc_rpl_dodag_t *dodag)
 
     dodag->dtsn++;
 
-    if (dodag->my_rank != GNRC_RPL_INFINITE_RANK) {
-        dodag->my_rank = GNRC_RPL_INFINITE_RANK;
-        trickle_reset_timer(&dodag->trickle);
-        gnrc_rpl_cleanup_start(dodag);
+
+    if ((CONFIG_GNRC_RPL_DODAG_FLOAT_TIMEOUT <= 0) || !_float_dodag(dodag)) {
```

Comment:
```suggestion
    if ((CONFIG_GNRC_RPL_DODAG_FLOAT_TIMEOUT <= 0) || !_float_dodag(dodag)) {
```

To fix the static error.